revops/index.html

```diff
@@ -52,47 +52,104 @@ <h2>Let's start building your Business&hellip;</h2>
     <div class="section-title-group">
       <h2 class="section-heading">Our Expertise</h2>
       <div class="section-subheading">
-        ➟ Are you struggling to align your sales, marketing and customer success
-        strategies? Or does your company need expert assistance with CRM
-        implementation? Or perhaps you are looking for a fractional executive to
-        fill knowledge gaps without the cost of a full time hire? We deliver
-        custom strategies that accelerate growth and improve operational
-        efficiency.
-        <a href="#contact">Start the conversation</a>
+        <p>
+          ➟ Revenue Operations (RevOps) isn’t magic, and while your company may
+          already be practicing aspects of RevOps, without a dedicated RevOps
+          professional, you might not be maximizing your potential.
+        </p>
+        <p>That’s where our expertise can help make a difference.</p>
+      </div>
+    </div>
+    <div class="section-title-group">
+      <h2 class="section-heading">Our Services</h2>
+      <div class="section-subheading">
+        <p>
+          ➟ Are you struggling to align your sales, marketing and customer
+          success strategies? Or does your company need expert assistance with
+          CRM implementation? Or perhaps you are looking for a fractional
+          executive to fill knowledge gaps without the cost of a full time hire?
+          We deliver custom strategies that accelerate growth and improve
+          operational efficiency.
+        </p>
+        <p>
+          Our Full Stack RevOps Services focus on the metrics that matter most
+          to you and by leveraging integrated technologies, like CRM systems,
+          automations and more, we streamline your business processes, create
+          efficient workflows, and drive sustainable growth - saving you
+          valuable time and resources.
+        </p>
+        <p>
+          Whether you are starting from scratch or looking to build your RevOps
+          framework from the ground up, our expert team integrates seamlessly
+          with your business. We hit the ground running and look to deliver
+          results from day one.
+          <a href="#contact">Start the conversation</a>
```

Comment:
How about slightly colder blue and weight increase for links?

![link1](https://github.com/user-attachments/assets/0061d2ec-a5eb-4d81-833f-cea26abd4cfd)
![link2](https://github.com/user-attachments/assets/c700f4b4-cdc6-4351-b013-a6684e65555d)
![link3](https://github.com/user-attachments/assets/c5e4fa5c-c5d0-4028-8f6a-6ba6b5dd9fe9)

Perhaps it would make sense to make button text the same color and weight as links?